src/item_pocket.cpp

```diff
@@ -0,0 +1,580 @@
+        }
+        if( data->rigid ) {
+            info.emplace_back( "DESCRIPTION", _( "This pocket is <info>rigid</info>." ) );
+        }
+        if( data->min_item_volume > 0_ml ) {
+            info.emplace_back( "DESCRIPTION",
+                               _( string_format( "Minimum volume of item allowed: <neutral>%s</neutral>",
+                                                 vol_to_string( data->min_item_volume ) ) ) );
+        }
+        info.emplace_back( "DESCRIPTION",
+                           _( string_format( "Volume Capacity: <neutral>%s</neutral>",
+                                             vol_to_string( data->max_contains_volume ) ) ) );
+        info.emplace_back( "DESCRIPTION",
+                           _( string_format( "Weight Capacity: <neutral>%s</neutral>",
+                                             weight_to_string( data->max_contains_weight ) ) ) );
+
+        info.emplace_back( "DESCRIPTION",
+                           _( string_format( "This pocket takes <neutral>%d</neutral> base moves to take an item out.",
+                                             data->moves ) ) );
+
+        if( data->watertight ) {
+            info.emplace_back( "DESCRIPTION",
+                               _( "This pocket can <info>contain a liquid</info>." ) );
+        }
+        if( data->gastight ) {
+            info.emplace_back( "DESCRIPTION",
+                               _( "This pocket can <info>contain a gas</info>." ) );
+        }
+        if( data->open_container ) {
+            info.emplace_back( "DESCRIPTION",
+                               _( "This pocket will <bad>spill</bad> if placed into another item or worn." ) );
+        }
+        if( data->fire_protection ) {
+            info.emplace_back( "DESCRIPTION",
+                               _( "This pocket <info>protects its contents from fire</info>." ) );
+        }
+        if( data->spoil_multiplier != 1.0f ) {
+            info.emplace_back( "DESCRIPTION",
+                               string_format(
+                                   _( "This pocket makes contained items spoil at <neutral>%.0f%%</neutral> their original rate." ),
+                                   data->spoil_multiplier * 100 ) );
+        }
+        if( data->weight_multiplier != 1.0f ) {
+            info.emplace_back( "DESCRIPTION",
+                               string_format( _( "Items in this pocket weigh <neutral>%.0f%%</neutral> their original weight." ),
+                                              data->weight_multiplier * 100 ) );
+        }
+    }
+}
+
+void item_pocket::contents_info( std::vector<iteminfo> &info, int pocket_number,
+                                 bool disp_pocket_number ) const
+{
+    const std::string space = "  ";
+
+    insert_separation_line( info );
+    if( disp_pocket_number ) {
+        info.emplace_back( "DESCRIPTION", _( string_format( "<bold>Pocket %d</bold>", pocket_number ) ) );
```

Comment:
```suggestion
        info.emplace_back( "DESCRIPTION", string_format( _( "<bold>Pocket %d</bold>", pocket_number ) ) );
```

Translation via `_()` can not be applied to formatted strings. This problem exists for other strings here as well.